app/src/main/java/org/simple/clinic/enterotp/EnterOtpEffectHandler.kt

```diff
@@ -45,6 +45,7 @@ class EnterOtpEffectHandler @AssistedInject constructor(
         .addAction(ShowNetworkError::class.java, uiActions::showNetworkError, schedulers.ui())
         .addAction(ShowUnexpectedError::class.java, uiActions::showUnexpectedError, schedulers.ui())
         .addTransformer(LoadOtpEntryProtectedStates::class.java, loadOtpEntryStates())
+        .addConsumer(ResetOtpAttemptLimit::class.java, { bruteForceProtection.resetFailedAttempts() }, schedulers.io())
```

Comment:
Since we are not expecting any data from the effect here, we can use `addAction` instead.

```suggestion
addAction(ResetOtpAttemptLimit::class.java, bruteForceProtection::resetFailedAttempts, schedulers.io())
```

Reply:
Ah right, my bad.